modules/Emails/include/displayEmailAddressOptInField.php

```diff
@@ -52,8 +52,15 @@
  */
 function displayEmailAddressOptInField($focus, $field, $value, $view)
 {
-    if(empty($focus->id) && empty($focus->from_name)) {
+    $addressField = 'from_name';
```

Comment:
Why the `$addressField` variable necessary here? It seems can not be anything else.

Reply:
There is a case in the emails module where it will use the from_addr instead of the from_name. This variable "configures" which field to use.